papaya-web/src/main/java/pl/edu/agh/papaya/notification/NotificationService.java

```diff
@@ -135,6 +140,14 @@ private void sendDeclarationsOpenOrReminderNotifications(LocalDateTime currentTi
         });
     }
 
+    private List<User> getUsers(List<UserInProject> usersInProject) {
```

Comment:
Nie dałoby się tego upchnąć w jakiejś innej klasie? `UserService`? `UserInProjectService`? Nie wiem, to mi wygląda jak jedna z tych metod, które zaraz skończą z egzemplarzem w każdej klasie

~isuseradminuseradminroleadminuserhasroleadmin~

Reply:
zmieniłem przekazywanie UserInProject na Userów bo UserInProject było kompletnie bez sensu